### ISSUE TITLE
multiple files upload in dev console

### ISSUE THREAD

**Issue by yureckey** [nhost/nhost#401]
Hi Team, in the files section of Developer console I can upload single file. I'm trying to transfer data from old (not hasura-based) system to Nhost, so I need to upload like a hundred files, is there a quick way to do it?

**Comment by elitan** [comment]
@szilarddoro Could we allow to select multiple files in the Nhost Storage UI?

**Comment by szilarddoro** [comment]
Yes, it's definitely something that we can allow. 👍🏼 I'll create the relevant issue for this.

**Comment by yureckey** [comment]
any update / estimation on this?

**Comment by elitan** [comment]
Nothing yet. We'll make sure to update here as soon as we have an update.
